lib/__tests__/index-test.js

```diff
@@ -240,7 +267,7 @@ describe('labels', () => {
   });
 
   after(() => {
-    React.createElement = createElement;
+    a11y.restore(React);
```

Comment:
are there any tests that stub *two* copies of React, and `restore` one of them, and assert that the other one is still stubbed?